### PR TITLE
Continuous release

### PR DESCRIPTION
**Pull request by marcauberer**
- Restructure endpoints
- Add certain delete endpoints
- Add Postman collection for testing
- Dependency upgrades
- Bugfixes

### CODE REVIEW

**Review by Ordinateur-Hack — APPROVED** [review]

**Comment by marcauberer** [comment]
Lets wait for the changes by @Ordinateur-Hack before we merge this one ...

**Comment by marcauberer** [comment]
@Ordinateur-Hack @Niklas-23 please review this pr and merge it, when you are ready ...

**Pull request merged**
